docs/topics/development/testing.rst

```diff
@@ -75,6 +73,16 @@ If you wish to re-run only the tests failed from the previous run::
 
     make test_failed
 
+Selenium Integration Tests
+--------------------------
+To run the selenium based tests outside of the docker container use the following command::
+
+    docker-compose exec --user root selenium-firefox tox -e ui-tests
+
+WARNING: This will WIPE the database as the test will create specific data for itself to look for.
+If you have anything you don't want to be deleted, please do not run these tests.
+
+For more detailed information on the integration tests, please see the Readme within the ``tests/ui`` directory.
```

Comment:
We should really add something that allows switching the database during these runs but that's just error-prone given that we'd have to stop the server, reconfigure, start again, wait for it to start up, run tests, shut down, reconfigure etc… 

Hmmm… Anyhow, a warning is perfect for now!

At some point we should revisit the idea and patch I had with the API-based database dump but one step at a time.